server/README.md

```diff
@@ -0,0 +1,181 @@
+# Attestation Server
+
+Attestation Server is a user space application for attestation purpose. 
+It receives and verifies the TEE's attestation [Evidence](https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md#attestation) and responses the corresponding [Attestation Results](https://github.com/confidential-containers/attestation-service/issues/1) to ensure the Evidence's generation environment is a real TEE environment and it's Trusted Computing Base (TCB) status are as expected.
+
+Attestation Server supports the following types of TEE:
+- SGX
+- TDX
+- SEVSNP
+- SAMPLE: Dummy TEE which is used to test/demo the Server's functionalities.
+
+Attestation Server depends on [Open Policy Agent (OPA)](https://www.openpolicyagent.org/docs/latest/) to evaluate the Evidence's TCB status during attestation. Each supported TEE's OPA `Policy(.rego)` and `Reference Data(.json)` can be customized in order to evaluate the TCB status precisely.
+Note: Please refer [Policy Language](https://www.openpolicyagent.org/docs/latest/policy-language/) for more about `.rego` syntax information.
+
+## API
+
+Attestation Server provides two groups of GRPC endpoints:
+- attestation: Attest the received Evidence which is defined in [protobuf](https://github.com/confidential-containers/attestation-service/server/proto/attestation.proto).
+- management: Customize the Server's configuration which is defined in [protobuf](https://github.com/confidential-containers/attestation-service/server/proto/management.proto).
+
+### Attestation
+
+The `attestation` group only includes one `Attestation` GRPC endpoint currently. It receives the attestation [Evidence](https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md#attestation), executes attestation procedures, and responses the [Attestation Results](https://github.com/confidential-containers/attestation-service/issues/1).
+
+`Attestation` message:
+```PROTO
+message AttestationRequest {
+    // Attestation evidence: https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md#attestatio
+    bytes evidence = 1;
```

Comment:
This is the full evidence message (including the nonce and public key) and not just the `tee-evidence` field, right?

Reply:
Yes, this is the full evidence message. I followed your previous suggestion that KBS should relay the evidence payload to AS directly. And the `tee-evidence` will only be parsed inside attestation service Crate internally.